README.md

```diff
@@ -3,7 +3,7 @@
 Contributors: lukecarbis, ryankienstra, Stino11, rheinardkorf
 Tags: Gutenberg, Blocks
 Requires at least: 5.0
-Tested up to: 5.2
+Tested up to: 5.3
```

Comment:
I think it'd be good to fix #456 before bumping this.

Reply:
Also, the Multi-select looks strange in WP 5.3-RC2:

<img width="651" alt="mult-select" src="https://user-images.githubusercontent.com/4063887/67652730-db41f480-f90b-11e9-9c6b-7fdd44727085.png">